CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+
+
+## 1.2.0
+
+`2020-07-21`
+
+- 🔥 支持`MongoDB`
+
+### 注意事项
+
+- 使用方需要引入MongoDB驱动包
+
+
+## 1.1.1
+
+`2020-07-13`
+
+- 🔥 支持`Redis`，基于`Lettuce`
+
+
+### 注意事项
```

Comment:
Codacy found an issue: [[no-duplicate-headings] Do not use headings with similar content (9:1)](https://app.codacy.com/gh/power4j/sequence/pullRequest?prid=5928643)